src/main/scala/dev/atedeg/UbiquitousScaladocPlugin.scala

```diff
@@ -1,7 +1,9 @@
 package dev.atedeg
 
-import sbt.{ AutoPlugin, Setting }
+import sbt.Keys.baseDirectory
+import sbt.{ AutoPlugin, Setting, Task }
 
+@SuppressWarnings(Array("org.wartremover.warts.Any", "org.wartremover.warts.Nothing"))
```

Comment:
This suppress warning is really necessary? If so, why?

Reply:
I really have no idea why but using `:=` seemed to always trigger the compiler to infer an `Any` or `Nothing` type no matter how many type annotations I used